artemis-server/src/main/java/org/apache/activemq/artemis/core/replication/ReplicationEndpoint.java

```diff
@@ -80,13 +80,17 @@
 import org.apache.activemq.artemis.core.server.cluster.qourum.SharedNothingBackupQuorum;
 import org.apache.activemq.artemis.core.server.impl.ActiveMQServerImpl;
 import org.apache.activemq.artemis.core.server.impl.SharedNothingBackupActivation;
+import org.jboss.logging.Logger;
 
 /**
  * Handles all the synchronization necessary for replication on the backup side (that is the
  * backup's side of the "remote backup" use case).
  */
 public final class ReplicationEndpoint implements ChannelHandler, ActiveMQComponent {
 
+   private final Logger logger = Logger.getLogger(ReplicationEndpoint.class);
```

Comment:
Nope.. I want to keep this.. it was really useful.. and I am thinking about adding more like this